### DISCUSSION TITLE
MTLS: Using .p12 Certs when connecting

### DISCUSSION THREAD

**Discussion by skyboard89** [victorbalssa/abacus#338] [General]
Hi,

My setup uses mTLS through Nginx to access the server. Therefore, my iPhone has a .p12 certificate.

When connecting through Safari, everything works fine. However, when using Abacus, it seems the connection fails as the app does not use the certificate to authenticate.

Is there a plan to add this feature?

Best Regards
sky

**Comment by victorbalssa** [comment]
Hey @skyboard89,

Currently Abacus won't work with custom certificates, we can keep it as a feature request for anyone who is interested in working on it.

Thanks!
